tests/Aql/AqlBuildSupervisedAqlValueTest.cpp

```diff
@@ -0,0 +1,196 @@
+#include "gtest/gtest.h"
+#include "Basics/ResourceUsage.h"
+#include "Basics/SupervisedBuffer.h"
+#include "Aql/AqlValue.h"
+#include <velocypack/Builder.h>
+#include <string>
+
+using arangodb::GlobalResourceMonitor;
```

Comment:
This is fine,
However we often use using namespace arangodb;
using namespace arangodb::aql;

It is a bit less to read.
but really optional comment here.